lib/id3c/api/schemas.py

```diff
@@ -100,3 +100,43 @@
     ],
     "additionalProperties": False
 }
+
+POST_INCIDENT_SCHEMA = {
+    "type": "object",
+    "properties": {
+        "sample_id": {
+            "type": "string",
+            "minLength": 8,
+            "maxLength": 8
+        },
+        "collection_id": {
+            "type": "string",
+            "minLength": 8,
+            "maxLength": 8
+        },
```

Comment:
To be processed by the manifest ETL, these keys should be `sample` and `collection`.